src/Raven.Studio/wwwroot/App/views/manage/clientConfiguration.html

```diff
@@ -8,30 +8,49 @@
+                                       data-bind="textInput: model.identityPartsSeparator, enable: _.includes(model.isDefined(), 'identityPartsSeparator'), 
+                                                  attr: { placeholder: model.separatorPlaceHolder }" />
+                            </div>
+                        </div>
+                    </div>
                     <div class="panel padding">
                         <div class="row">
                             <div class="col-sm-6">
                                 <div class="checkbox">
                                     <input id="checkboxMaxRequests" type="checkbox"
-                                           data-bind="checked: model.isDefined, checkedValue: 'maxNumberOfRequestsPerSession', validationElement: model.maxNumberOfRequestsPerSession">
+                                           data-bind="checked: model.isDefined, checkedValue: 'maxNumberOfRequestsPerSession'" />
                                     <label for="checkboxMaxRequests">
                                         Maximum number of requests per session
                                     </label>
                                 </div>
                             </div>
                             <div class="col-sm-4" data-bind="validationElement: model.maxNumberOfRequestsPerSession">
-                                <input class="form-control" type="number" min="0" placeholder="Enter requests number"
-                                       data-bind="numericInput: model.maxNumberOfRequestsPerSession, enable: _.includes(model.isDefined(), 'maxNumberOfRequestsPerSession')" />
+                                <input class="form-control" type="number" min="0"
+                                       data-bind="numericInput: model.maxNumberOfRequestsPerSession, enable: _.includes(model.isDefined(), 'maxNumberOfRequestsPerSession'), 
+                                                  attr: { placeholder: model.requestsNumberPlaceHolder }" />
                             </div>
                         </div>
                     </div>
                     <div class="panel padding">
                         <div class="row">
                             <div class="col-sm-6">
                                 <div class="checkbox">
-                                    <input id="useSessionContextToggle" type="checkbox" 
-                                           data-bind="checked: model.isDefined, checkedValue: 'useSessionContextForLoadBehavior', disable: _.includes(model.isDefined() , 'readBalanceBehavior')">
-                                    <label for="useSessionContextToggle">
+                                    <input id="checkboxUseSessionContext" type="checkbox"
+                                           data-bind="checked: model.isDefined, checkedValue: 'useSessionContextForLoadBehavior', disable: _.includes(model.isDefined() , 'readBalanceBehavior')" />
```

Comment:
Didn't we agree that we don't want to disable this option? But when we choose this option we uncheck second toggle.

Reply:
fixed